api_app/analyzers_manager/observable_analyzers/dns/dns_resolvers/quad9_dns_resolver.py

```diff
@@ -20,23 +18,33 @@ class Quad9DNSResolver(classes.ObservableAnalyzer):
     query_type: str
 
     def run(self):
-        try:
-            observable = self.observable_name
-            # for URLs we are checking the relative domain
-            if self.observable_classification == "url":
-                observable = urlparse(self.observable_name).hostname
-
-            headers = {"Accept": "application/dns-json"}
-            url = "https://dns.quad9.net:5053/dns-query"
-            params = {"name": observable, "type": self.query_type}
-
-            quad9_response = requests.get(url, headers=headers, params=params)
-            quad9_response.raise_for_status()
-            resolutions = quad9_response.json().get("Answer", [])
-        except requests.RequestException:
-            raise AnalyzerRunException(
-                "an error occurred during the connection to Quad9"
-            )
+        observable = self.observable_name
+        # for URLs we are checking the relative domain
+        if self.observable_classification == "url":
+            observable = urlparse(self.observable_name).hostname
+
+        url = "https://dns.quad9.net:5053/dns-query"
+        headers = {"Accept": "application/dns-json"}
```

Comment:
This should be class attributes (and `url` should be named `base_url` if i remember correctly) so that the healthcheck could work automatically

Reply:
I made the var a class attr. I checked the code about the health check and should work unfortunately a head request to the url get a 415 as response (I also tried removing the port or just the base url ans didn't work, always a > 400 status code).